apps/chat/src/components/Common/Forms/TooltipWrapper.tsx

```diff
@@ -0,0 +1,28 @@
+import { ComponentType, forwardRef } from 'react';
+
+import Tooltip from '@/src/components/Common/Tooltip';
+
+interface TooltipWrapperProps {
+  tooltip?: string;
+}
+
+export function TooltipWrapper<T extends object, R>(
+  Component: ComponentType<T>,
+) {
+  const EnhancedComponent = forwardRef<R, TooltipWrapperProps & T>(
+    ({ tooltip, ...props }, ref) => (
+      <div>
+        {tooltip && (
```

Comment:
this logic already exists into `Tooltip` component
![image](https://github.com/user-attachments/assets/0cb2d33b-5191-4b84-bf57-e88bbddf615a)